diff --git a/dist/index.hbs b/dist/index.hbs
index 10332d5f0ae30ac6a486896750f59cfbb1b80100..941ece57da163135ddf0f2e88515941a35612720 100644
--- a/dist/index.hbs
+++ b/dist/index.hbs
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>{{title}}</title>
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet">
+  {{!-- <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet"> --}}
   <link rel="stylesheet" href="{{swaggerCdnUrl}}/{{swaggerVersion}}/swagger-ui.min.css">
   <link rel="icon" type="image/png" href="{{favicon}}" />
   <style>
